docs/spec/overview.md

```diff
@@ -0,0 +1,80 @@
+# Resource Groups
+
+Knative Eventing API is grouped into _Channels_ and _Feeds_:
+
+* _Channels_ define an abstraction between the eventing infrastructure and the
+  consumption and production of the events.
+
+* _Feeds_ bridge the event source into the eventing framework.
+
+* _Flows_ are the higher order abstractions for eventing that use _Channels_
```

Comment:
I'd prefer to talk about what Flows gives the user, rather than where it sits in relation to other objects.

Reply:
I am at a loss for the correct words for this

Reply:
the desired path from an external Source of events to a destination that will react to the events

?

Reply:
not strictly "external", e.g. the `k8sevents` EventSource

Reply:
I am trying: `* _Flows_ abstract the path of an event from a source reaches an event consumer.
`